lib/Event/Traits/DisableEventTrait.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Event\Traits;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+trait DisableEventTrait
+{
+
+    /**
+     * @var array
+     */
+    protected static $disabledEvents = [];
```

Comment:
```suggestion
    private $activeDispatchingEvents = [];
```
I guess we need to consider this here as well then? 
https://github.com/pimcore/pimcore/blob/9859aacdc0653090182c1d403bcc4b60226bf4e8/models/Element/AbstractElement.php#L391